index.bs

```diff
@@ -390,6 +390,74 @@ by a custom configuration.
 {}
    ```
 
+# Security Considerations {#security-considerations}
+
+The Sanitizer API is intended to prevent DOM-Based Cross-Site Scripting
+by traversing a DocumentFragment and removing elements and attributes
+according to a configuration. The specified API must not support
+the construction of a Sanitizer object that leaves script-capable markup in
+and doing so would be a bug in the threat model.
+
+That being said, there are security issues which the correct usage of the
+Sanitizer API will not be able to protect against and the scenarios will be
+laid out in the following sections.
+
```

Comment:
Editorial: This is admittedly super totally redundant with your introductory paragraph, but given that pretty much everyone asks about it I'd still add an non-overridable defaults as a separate item.


Maybe something like:

## Safe by Default

The Sanitizer uses a safe-by-default allow list. Additionally, the Sanitizer has non-overridable baseline behaviour for elements and attributes the browser knows to contain script. For example, it's not possible to configure a Sanitizer to allow a `<script>` element to pass through.